plugins/httpapi/jsonrpc.py

```diff
@@ -9,8 +9,8 @@
 
 DOCUMENTATION = """
 ---
-author: fischbacher.markus@gmail.com
-httpapi : zabbix
+name: zabbix
```

Comment:
```suggestion
name: jsonrpc
```
Same here: the plugin's name is `jsonrpc`, not `zabbix`.